plugins/package-managers/pub/src/main/kotlin/Pub.kt

```diff
@@ -750,7 +750,7 @@ class Pub(
     private fun containsFlutterSdk(workingDir: File): Boolean {
         val pubspec = parsePubspec(workingDir.resolve(PUBSPEC_YAML))
 
-        return (pubspec.dependencies.orEmpty()["flutter"] as? Pubspec.SdkDependency)?.sdk == "flutter"
+        return pubspec.dependencies.orEmpty().values.any { (it as? Pubspec.SdkDependency)?.sdk == "flutter" }
```

Comment:
Commit message:
- "Take" -> "Make"
- Please explain *why* that is better and safe to do. Maybe @MarcelBochtler can help to find an argument here?

Reply:
You can have a dependency on `flutter_driver` (IIRC), which is part of the flutter SDK.

Reply:
In one of our lockfile for example we have:
```
  sky_engine:
    dependency: transitive
    description: flutter
    source: sdk
    version: "0.0.99"
```

Reply:
I've split out this commit to https://github.com/oss-review-toolkit/ort/pull/9082.